sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOverrides.scala

```diff
@@ -2245,9 +2245,7 @@ object GpuOverrides extends Logging {
       ExprChecks.fullAgg(
         TypeSig.LONG, TypeSig.LONG,
         repeatingParamCheck = Some(RepeatingParamCheck(
-          "input", _gpuCommonTypes + TypeSig.DECIMAL_128 +
-              TypeSig.STRUCT.nested(_gpuCommonTypes + TypeSig.DECIMAL_128),
-          TypeSig.all))),
+          "input", TypeSig.all, TypeSig.all))),
```

Comment:
Do we really want to say _every_ time here?  In the past I believe @revans2 had some reservations about stating support for things like UDTs, and there are types we're saying we support here that are not being tested in the integration tests.

Reply:
Bobby said: we should be able to make this work for any data type. 
https://github.com/NVIDIA/spark-rapids/issues/5060#issuecomment-1079423319